packages/edit-site/src/components/page-pages/index.js

```diff
@@ -31,6 +31,7 @@ import {
 import SideEditor from './side-editor';
 import Media from '../media';
 import { unlock } from '../../lock-unlock';
+import { ENUMERATION_TYPE, OPERATOR_IN } from '../dataviews/constants';
```

Comment:
This import and the next one assume that the future "data views" package would export these constants as well. I'm not sure I like exporting constants from a package to another but I guess when we extract the package we can either duplicate these constants in edit-site as well or replace them with strings.